crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -398,7 +398,14 @@ def check_for_running_instances
           return
         end
 
-        cmd = "openstack server list --all-projects --long " \
+        insecure = ""
+        keystone = ::Node.find("run_list_map:keystone-server").first
+        if keystone[:keystone][:ssl][:insecure] ||
+            (controller[:nova][:ssl][:enabled] && controller[:nova][:ssl][:insecure])
+          insecure = "--insecure"
+        end
```

Comment:
This block looks weird to me, Don't we have something better in crowbar app to realize we have insecure setup?
Or should I rather check proposals then nodes? @rhafer , @vuntz ?